client/src/components/Form/Form.js

```diff
@@ -1,43 +1,74 @@
-import React, { useState, useEffect } from "react";
+import React, { useState, useEffect } from 'react';
+import clsx from 'clsx';
 
 // Material-UI Imports
-import { createStyles, makeStyles } from "@material-ui/core/styles";
-import Button from "@material-ui/core/Button";
-import InputLabel from "@material-ui/core/InputLabel";
-import MenuItem from "@material-ui/core/MenuItem";
-import FormControl from "@material-ui/core/FormControl";
-import Select from "@material-ui/core/Select";
+import { createStyles, makeStyles } from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
 
 // Local imports
-import DistanceSlider from "./DistanceSlider";
-import CheckboxGroup from "./CheckboxGroup";
+import DistanceSlider from './DistanceSlider';
+import CheckboxGroup from './CheckboxGroup';
+
+import data from '../../data/demo.json';
+
+function haversineCalc(la1,la2,lo1,lo2){
+  const R = 6371e3; // meters
+  const φ1 = (la1 * Math.PI) / 180; // φ, λ in radians
+  const φ2 = (la2 * Math.PI) / 180;
+  const Δφ =
+    ((la2 - la1) *
+      Math.PI) /
+    180;
+  const Δλ =
+    ((lo2 - lo1) *
+      Math.PI) /
+    180;
+  const a =
+    Math.sin(Δφ / 2) * Math.sin(Δφ / 2) +
+    Math.cos(φ1) * Math.cos(φ2) * Math.sin(Δλ / 2) * Math.sin(Δλ / 2);
+  const c = 2 * Math.atan2(Math.sqrt(a), Math.sqrt(1 - a));
+
+  return R * c * 0.000621371; // in miles
+}
 
 const useStyles = makeStyles((theme) =>
   createStyles({
     formRowOne: {
-      display: "flex",
-      justifyContent: "space-around",
+      display: 'flex',
+      justifyContent: 'space-around',
+      [theme.breakpoints.down('xs')]: {
+        flexDirection: 'column',
+      },
     },
     formControl: {
       margin: theme.spacing(1),
       minWidth: 120,
     },
+    delivery: {
+      [theme.breakpoints.down('xs')]: {
+        order: 1,
+      },
+    },
     selectEmpty: {
       marginTop: theme.spacing(2),
     },
     btnGrp: {
-      display: "inline-grid",
+      display: 'inline-grid',
     },
     clearBtn: {
-      fontSize: "0.7rem",
+      fontSize: '0.7rem',
     },
   })
 );
 
-export default function Form({ setResults }) {

```

Comment:
I renamed this in my PR, but lets be more descriptive with component names, we don't want to have to look at code to determine what kind of form it is.